recipes/carbonplan-data/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "carbonplan-data" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/carbonplan-data-{{ version }}.tar.gz
+  sha256: 8f1fea3701d099b25741f7828c27f2b3bfbbaf7e0abfdd5cbb0030c058303dbc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools-scm
+    - setuptools >=30.3.0
+    - pip
+  run:
+    - python >=3.7
+    - carbonplan
+    - urlpath
+    - wget
```

@rsignell, looks like they fixed the lack of intake in the requirement metadata in 0.4.1, see https://github.com/carbonplan/data/blob/c8261129305c08cf841fd24f8ee2e5ddae424f44/requirements.txt#L2C1-L2C7

You can  either update this recipe or add it here manually.

```suggestion
    - intake
    - python-wget
```

PS: python-wget is the PyPI wget package, b/c conda is language agnostic we had to rename it to avoid clashing with the wget package.